src/additions/test_test_generators.py

```diff
@@ -0,0 +1,35 @@
+"""Generators.
+
+@see: https://www.learnpython.org/en/Generators
+
+Generators are used to create iterators, but with a different approach. Generators are simple
+functions which return an iterable set of items, one at a time, in a special way.
+
+When an iteration over a set of item starts using the for statement, the generator is run. Once the
+generator's function code reaches a "yield" statement, the generator yields its execution back to
+the for loop, returning a new value from the set. The generator function can generate as many
+values (possibly infinite) as it wants, yielding each one in its turn.
+"""
+
+import random
+
+
+def even_number_generator():
```

Comment:
This method should have an argument to use as number of integers to generate and then we can use this method to generated a large list of of number without consuming too much memory.
https://towardsdatascience.com/pythons-list-generators-what-when-how-and-why-2a560abd3879

Reply:
suggestion file uploaded with this task